src/test/librbd/cache/test_mock_ReplicatedWriteLog.cc

```diff
@@ -47,14 +46,15 @@ inline ImageCtx *get_image_ctx(MockImageCtx *image_ctx) {
 
 namespace librbd {
 namespace cache {
+namespace pwl {
 
 using ::testing::_;
 using ::testing::DoDefault;
 using ::testing::InSequence;
 using ::testing::Invoke;
 
 struct TestMockCacheReplicatedWriteLog : public TestMockFixture {
-  typedef WriteLogCache<librbd::MockImageCtx> MockReplicatedWriteLog;
+  typedef librbd::cache::pwl::ReplicatedWriteLog<librbd::MockImageCtx> MockReplicatedWriteLog;
```

Comment:
This type should be `AbstractWriteLog`. I don't think the RWL UT will pass with this reference?